torch/_inductor/codegen/cuda/cuda_kernel.py

```diff
@@ -298,14 +298,18 @@ def __init__(
         layout: Layout,
         make_kernel_render: Callable[[CUDATemplateBuffer, Optional[List[IRNode]]], str],
         bmreq: CUDABenchmarkRequest,
-        template: "CUDATemplate",
+        template: "CUDATemplate",  # type: ignore[name-defined]
```

Comment:
Huh, this type ignore shouldn't be necessary...

Reply:
I add these type ignores only on linter errors when I run lintrunner locally. I also have the impression that these are a bit inconsistent, e.g. sometimes I had linter warnings about code that was just fine the day before.